ml/v1/ml.proto

```diff
@@ -59,7 +59,36 @@ service MLService {
             body: "*"
         };
     }
+    
+    // List the available size configurations for MLServices.
+    // Note that the returned size specifications are applied for ML Jobs.
+    // Required permissions:
+    // - ml.mlservicessize.list on the deployment (if deployment_id is provided)
+    // - None, authenticated only
+    rpc ListMLServicesSizes(ListMLServicesSizesRequest) returns (MLServicesSizeList);
+}
+
+// List of MLServicesSize.
+message MLServicesSizeList {
+    // Items in this list.
+    repeated MLServicesSize items = 1;
+}
+
+message ListMLServicesSizesRequest {
+    // Optional ID of the Deployment for which sizes are being requested.
+    // If set, the response will exclude any sizes that are unavailable for the specified deployment model.
+    string deployment_id = 1;
+}
 
+// MLServicesSize represents the resources allocated for MLServices.
+// Note that the specified configuration is applied for the ML jobs.
+message MLServicesSize {
+    // Identifier of the size configuration.
+    string size_id = 1;
+    // Amount of CPU allocated (in vCPU units)
+    float cpu = 2;
```

Comment:
I would like to see the gpu setting here as well

Reply:
Will be added in a follow-up PR, since we currently don't have a way to internally schedule the job pods based on the GPU memory size (so once that's in place, I will add a new field here)

Reply:
Added, as per our Slack conversation